File: MachineWatcher/Model/Drive.cs

using System.Runtime.Serialization;

namespace MachineWatcher.Model
{
	[DataContract]
	public class Drive
	{
		[DataMember(Name = "name")]
		public string Name { get; set; }
		[DataMember(Name = "availableSpace")]
		public long AvailableSpace { get; set; }
		[DataMember(Name = "totalSpace")]
		public long TotalSpace { get; set; }
	}
}

File: machine-manager/src/main/java/com/philipp/manager/model/Drive.java
package com.philipp.manager.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "drive")
public class Drive {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int id;

	@Column(nullable = false, length = 32)
	private String name;

	@Column(nullable = false)
	private long availableSpace;

	@Column(nullable = false)
	private long totalSpace;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public long getAvailableSpace() {
		return availableSpace;
	}

	public void setAvailableSpace(long availableSpace) {
		this.availableSpace = availableSpace;
	}

	public long getTotalSpace() {
		return totalSpace;
	}

	public void setTotalSpace(long totalSpace) {
		this.totalSpace = totalSpace;
	}
}
